### ISSUE TITLE
Remove `influxdb2_client`

### ISSUE THREAD

**Issue by crepererum** [influxdata/influxdb_iox#6944]
Users should use `influxdb_iox_client` instead.

Blocked on #6943.

**Comment by NiwakaDev** [comment]
I'd like to work on this issue. Can I?

**Comment by crepererum** [comment]
@NiwakaDev sure, but this is still blocked by #6943 so that should probably be fixed first. Do you wanna fix #6943?

**Comment by NiwakaDev** [comment]
>sure, but this is still blocked by https://github.com/influxdata/influxdb_iox/issues/6943 so that should probably be fixed first.

Oh, sorry, I missed it. I'll wait until #6943 is fixed.

>Do you wanna fix https://github.com/influxdata/influxdb_iox/issues/6943?

I'll address another issue.
